plugins/modules/mso_schema_template_anp_epg.py

```diff
@@ -147,6 +154,35 @@
     - Whether this EPG is added to preferred group or not.
     - When not specified, this parameter defaults to C(no).
     type: bool
+  qos_level:
+    description:
+    - Quality of Service (QoS) feature allows you to classify the network traffic in your fabric
+    -  and then to prioritize and police the traffic flow to help avoid congestion in your network.
```

Comment:
It helps prioritize and police the traffic flow to avoid congestion in the network.